ci_feature/main.sh

```diff
@@ -25,31 +25,9 @@ sudo setfacl -m user:omsagent:r /etc/kubernetes/host/azure.json
 # add permission for omsagent user to log folder. We also need 'x', else log rotation is failing. TODO: Investigate why.
 sudo setfacl -m user:omsagent:rwx /var/opt/microsoft/docker-cimprov/log
 
-DOCKER_SOCKET=/var/run/host/docker.sock
```

Comment:
for each file changed under ci_feature/ , the corresponding files under ci_feature_prod/ should be updated as well.

Reply:
sure. will make the change in ci_feature_prod/ as well.

Reply:
done.